tests/instances.tcl

```diff
@@ -187,17 +187,15 @@ proc log_crashes {} {
     set logs [glob */log.txt]
     foreach log $logs {
         set fd [open $log]
-        set found 0
         while {[gets $fd line] >= 0} {
             if {[string match $start_pattern $line]} {
                 puts "\n*** Crash report found in $log ***"
-                set found 1
-            }
-            if {$found} {
-                puts $line
+                puts [exec cat $log]
```

Comment:
There must be a better way to get the file contents than to exec the separate program `cat`. How about `seek` + `read`?

```suggestion
                seek $log 0
                puts [read $log]
```

Reply:
./runtest-cluster (or ./runtest-sentinel) does not have the dump-logs option, it is very old and we don't maintain it a lot. Most ot the cluster test are migrate to normal test suite, some are left in #2297, we may completely remove it in the future (maybe not possible, because there is still sentinel test..)

we doing this exec cat thing a lot, like:
```
proc dump_server_log {srv} {
    set pid [dict get $srv "pid"]
    puts "\n===== Start of server log (pid $pid) =====\n"
    puts [exec cat [dict get $srv "stdout"]]
    puts "===== End of server log (pid $pid) =====\n"

    puts "\n===== Start of server stderr log (pid $pid) =====\n"
    puts [exec cat [dict get $srv "stderr"]]
    puts "===== End of server stderr log (pid $pid) =====\n"
}
```

Reply:
> So log_crashes is not used in the normal runtest?

yes, i think so. Normal runtest has it own cleanup proc there.